conans/model/conf.py

```diff
@@ -90,7 +91,7 @@
     "tools.google.bazel:configs": "List of Bazel configurations to be used as 'bazel build --config=config1 ...'",
     "tools.google.bazel:bazelrc_path": "List of paths to bazelrc files to be used as 'bazel --bazelrc=rcpath1 ... build'",
     "tools.meson.mesontoolchain:backend": "Any Meson backend: ninja, vs, vs2010, vs2012, vs2013, vs2015, vs2017, vs2019, xcode",
-    "tools.meson.mesontoolchain:extra_machine_files": "List of paths for any additional native/cross file references to be appended to the existing Conan ones",
+    "tools.meson.mesontoolchain:extra_machine_files": "List of additional native/cross file references to be appended to the Meson setup command, e.g., ['--native-file myfile.ini']",
```

Comment:
Yep, this can be breaking, lets think if we can introduce it without breaking.

Reply:
Agree